### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Alhajideen**
A simple html-css project with a header saying "Hello Microverse"

### CODE REVIEW

File: .github/workflow/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```
Comment (MeqdamAlqudah) [inline .github/workflow/linters.yml]: - [ ] Please make sure that setup linters correctly 
image

File: .github/workflow/linters.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+on: pull_request
+
```
Comment (MeqdamAlqudah) [inline .github/workflow/linters.yml]: - [ ] Please make sure to add a good description to your pull request 
- [ ] Please the title of the pull request should describe a feature like feature/addHeading 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Alhajideen , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1:  Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflow/linters.yml (hunk 2 above)
Comment (Alhajideen) [reply .github/workflow/linters.yml]: Hi,
Thank you.
All changes have been effected.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Alhajideen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**